unit-tests/to_upper_lower.F90

```diff
@@ -0,0 +1,48 @@
+!> @file
+
+!> output
```

Comment:
Actually convert these to non-doxygen comments. All production code gets doxygen, but not test code. ;-)

ALso add your name as the author of this test.